helpers/parse_helpers.go

```diff
@@ -330,7 +319,14 @@ func CreateCryptoConfigWithOpts(recipients []string, keys []string, opts CryptoC
 		}
 		encryptCcs = append(encryptCcs, jweCc)
 
-		pkcs11Cc, err := encconfig.EncryptWithPkcs11(opts.Pkcs11Config, pkcs11Pubkeys, pkcs11Yamls)
+		var p11conf *pkcs11.Pkcs11Config
+		if len(pkcs11Yamls) > 0 {
+			p11conf, err = pkcs11config.GetUserPkcs11Config()
+			if err != nil {
+				return encconfig.CryptoConfig{}, err
+			}
+		}
+		pkcs11Cc, err := encconfig.EncryptWithPkcs11(p11conf, pkcs11Pubkeys, pkcs11Yamls)
```

Comment:
should this be moved into the if block above as well?

Reply:
Good point. There's something wrong with the EncryptWithPkcs11 function. We should be able to just have PEM keys (pkcs11Pubkeys) and no pkcs11 yaml files (with URIs) and in this case we then do not need p11conf, either. Let me have a look at that function.

Reply:
Fixed now.